docs/proposals/docker/docker_volume_facts.md

```diff
@@ -0,0 +1,48 @@
+
+# Docker_Volume_Facts Module Proposal
+
+## Purpose and Scope
+
+Docker_volume_facts will inspect volumes.
+
+Docker_volume_facts will use docker-py to communicate with either a local or remote API. It will
+support API versions >= 1.14. API connection details will be handled externally in a shared utility module similar
+to how other cloud modules operate.
+
+## Parameters
+
+Docker_volume_facts will accept the parameters listed below. API connection parameters will be part of a shared
+utility module as mentioned above.
+
+
```

Comment:
also, does this point at docker host or container?

do we need such fine grained facts module or does it make more sense to have docker_facts or inventory script already populate this info?